components/colorwheel/index.css

```diff
@@ -56,35 +95,69 @@
   }
 }
 
-.spectrum-ColorWheel-gradient {
-  position: relative;
-  z-index: 0;
-  width: var(--spectrum-colorwheel-width);
-  height: var(--spectrum-colorwheel-height);
-  border-radius: 100%;
-  box-sizing: border-box;
-
-  border-width: var(--spectrum-colorwheel-border-size);
-  border-style: solid;
-}
-
-.spectrum-ColorWheel-wheel {
-  position: relative;
-  z-index: 1;
+.spectrum-ColorWheel-internal{
+  display: flex;
+  position: absolute;
+  inset-inline-start: 0;
+  inset-inline-end: 0;
+  inset-block-start: 0;
+  inset-block-end: 0;
+  inline-size: var(--mod-colorwheel-colorarea-container-size, var(--spectrum-colorwheel-colorarea-container-size));
+  block-size: var(--mod-colorwheel-colorarea-container-size, var(--spectrum-colorwheel-colorarea-container-size));
+  margin: auto;
 }
 
-.spectrum-ColorWheel-outerCircle,
-.spectrum-ColorWheel-innerCircle {
-  fill: transparent;
-
-  stroke-width: var(--spectrum-colorwheel-border-size);
+.spectrum-ColorWheel-colorarea-container{
+  block-size: auto;
+  inline-size: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: var(--mod-colorwheel-colorarea-margin, var(--spectrum-colorwheel-colorarea-margin));
 }
 
 .spectrum-ColorWheel-slider {
   @extend %spectrum-ColorControl-hiddenField;
 }
 
 .spectrum-ColorWheel-handle {
-  left: 50%;
+  transform: translate(var(--mod-colorwheel-colorhandle-position, var(--spectrum-colorwheel-colorhandle-position)), 0);
   top: 50%;
+  left: 50%;
```

Comment:
I am using physical properties for the positioning of the ColorHandle. 
`top: 50%; left: 50%` Positions in the center regardless for direction, while ` inset-inline: 50%; inset-block: 50%;` or `inset-inline-start: 50%; inset-block-start` is only centered when the direction is `ltr`

Reply:
[codepen displaying what I was running into ](https://codepen.io/jenndiaz/pen/ZEMGOqg)

Reply:
might need an over-write here like 
```
.spectrum-ColorWheel-handle {
 [dir="rtl"] & {
```

Reply:
was able to use an over-write and get it working with logical properties.